assets/launcher/lang/sklmessages_tr_TR.properties

```diff
@@ -59,7 +59,7 @@ editor.visibility=Launcher Görünürlüğü
 editor.hide=Launcher gizle ve oyun başladığında yeniden göster
 editor.close=Oyun başladığında launcheri kapat
 editor.keep=Launcheri açık tut
-editor.snap=Ekran görüntüleri
+editor.snap=Snapshots
```

Comment:
@Meka4756 "Ekran görüntüleri" means "**screen**shots" or sth like that, while we have "**snap**shots" here. That's why I have brought back the English word.

Reply:
But generally we use **Snapshots** to say **Ekran Görüntüleri** in informal language. Thanks for correcting my mistake :)

Reply:
If you have a good replacement for the English "snapshots", you can just write it here, in the comment. I'll add it in the next commit :) If not, to stop me from waiting for nothing, use the relatively new GitHub feature and click on the "Resolve conversation" button below.

Reply:
It's called "test sürümü" in Minecraft and Minecraft Launcher.

Reply:
You're probably right, but this string was translated by @Meka4756 and because of that I don't want to write anything here, which is not in English, without a clear recommendation.